extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -143,68 +153,76 @@
-				var targetObject = $(this);
+			$themeTab
+				.find('.slider')
+				.find('li')
+				.click(function () {
+					var targetObject = $(this);
 
-				// highlight selected theme
-				targetObject.parent().find('.selected').removeClass('selected').end().end().addClass('selected');
+					// highlight selected theme
+					targetObject.parent().find('.selected').removeClass('selected').end().end().addClass('selected');
 
-				ThemeDesigner.set('theme', targetObject.attr('data-theme'));
-				if (ThemeDesigner.checkBgImageIsSet()) {
-					ThemeDesigner.loadImage(ThemeDesigner.settings['background-image']);
-				} else {
-					ThemeDesigner.checkBgIsDynamic(0);
-				}
-				ThemeDesigner.resetPageOpacity();
-			});
-
-			// select current theme
-			$('#ThemeTab').find('[data-theme=' + ThemeDesigner.settings.theme + ']').addClass('selected');
+					ThemeDesigner.set('theme', targetObject.attr('data-theme'));
+					if (ThemeDesigner.checkBgImageIsSet()) {
+						ThemeDesigner.loadImage(ThemeDesigner.settings['background-image']);
+					} else {
+						ThemeDesigner.checkBgIsDynamic(0);
+					}
+					ThemeDesigner.resetPageOpacity();
+				})
+				// select current theme
+				.find('[data-theme=' + ThemeDesigner.settings.theme + ']')
+				.addClass('selected');
 		},
 
 		customizeTabInit: function () {
-			$('#CustomizeTab').find('li').find('img[id*="color"]').click(function (event) {
-				ThemeDesigner.showPicker(event, 'color');
-			});
+			$('#CustomizeTab')
+				.find('li img[id*="color"]')
+				.click(function (event) {
+					ThemeDesigner.showPicker(event, 'color');
+				});
+
 			$('#swatch-image-background').click(function (event) {
 				ThemeDesigner.showPicker(event, 'image');
 			});
 
-			$('#tile-background').change(function () {
-				ThemeDesigner.checkTiledBg(ThemeDesigner.splitOption);
-				ThemeDesigner.set('background-tiled', $(this).attr('checked') ? 'true' : 'false');
-			});
-			$('#fix-background').change(function () {
-				ThemeDesigner.set('background-fixed', $(this).attr('checked') ? 'true' : 'false');
+			$('#swatch-community-header-image-background').click(function (event) {
+				ThemeDesigner.showPicker(event, 'community-header');
 			});
 
-			$('#tile-background').attr('checked', ThemeDesigner.settings['background-tiled'] === 'true');
-			$('#fix-background').attr('checked', ThemeDesigner.settings['background-fixed'] === 'true');
+			$('#tile-background')
+				.change(function () {
+					ThemeDesigner.checkTiledBg(ThemeDesigner.splitOption);
+					ThemeDesigner.set('background-tiled', $(this).attr('checked') ? 'true' : 'false');
+				})
+				.attr('checked', ThemeDesigner.settings['background-tiled'] === 'true');
+
+			$('#fix-background')
+				.change(function () {
+					ThemeDesigner.set('background-fixed', $(this).attr('checked') ? 'true' : 'false');
+				})
+				.attr('checked', ThemeDesigner.settings['background-fixed'] === 'true');
 
 			// TODO: Remove IF statement after fluid layout global release
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)